parsers/src/main/scala/uk/gov/ons/addressIndex/crfscala/CrfFeature.scala

```diff
@@ -34,34 +34,29 @@ trait CrfFeature[T] {
     * @return
     */
   def toCrfJniInput(input: CrfToken, next: Option[CrfToken] = None, previous: Option[CrfToken] = None): CrfJniInput = {
```

Comment:
I'd expect some really good Scala docs for a method this esoteric. There's nothing complex going on per say, but it's pretty hard to follow, speaking for myself as an external reader. I know it's already in the bucket list, and today is just a marvellous opportunity to outshine Dickens and earn the first Nobel prize for literature for a Scaladoc comment.